src/core/lib/security/credentials/oauth2/oauth2_credentials.cc

```diff
@@ -273,6 +280,7 @@ bool grpc_oauth2_token_fetcher_credentials::get_request_metadata(
     grpc_polling_entity* pollent, grpc_auth_metadata_context /*context*/,
     grpc_credentials_mdelem_array* md_array, grpc_closure* on_request_metadata,
     grpc_error** /*error*/) {
+  add_additional_metadata(md_array);
```

Comment:
naming nit: can we prefix this with `maybe`, i.e. change to `maybe_add_additional_metadata`

Reply:
Done